pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/parse/DynamicFieldsParser.java

```diff
@@ -78,12 +79,9 @@ protected void getProperDisplayTypeWithPossibleValues() {
         Optional<FieldSettingsDto> fieldSettingsByInstanceNameAndColumnName = getFieldSettingsByColumnName();
         if (fieldSettingsByInstanceNameAndColumnName.isPresent()) {
             FieldSettingsDto fieldSettings = fieldSettingsByInstanceNameAndColumnName.get();
-            displayType = StringUtils.isNotBlank(fieldSettings.getDisplayType())
-                    ? fieldSettings.getDisplayType()
-                    : StringUtils.EMPTY;
-            possibleValuesJson = StringUtils.isNotBlank(fieldSettings.getPossibleValues())
-                    ? fieldSettings.getPossibleValues()
-                    : StringUtils.EMPTY;
+            displayType = StringUtils.isNotBlank(fieldSettings.getDisplayType()) ? fieldSettings.getDisplayType() : StringUtils.EMPTY;
```

Comment:
I really don't like how checkstyle changes ternary condition, I think pattern: 
:
?
is more straightforward than: 
? :